src/main/java/uk/gov/ch/developer/docs/controller/AbstractPageController.java

```diff
@@ -7,13 +7,13 @@
 import uk.gov.ch.developer.docs.models.nav.NavBarModel;
 import uk.gov.ch.developer.docs.models.nav.NavBarModelBuilder;
 import uk.gov.ch.developer.docs.models.user.IUserModel;
-import uk.gov.ch.developer.docs.session.SessionService;
+import uk.gov.ch.oauth.session.SessionFactory;
```

Comment:
I don't think we should rely on OAUTH. It will make it more difficult to extract this library.

Reply:
I don't know why this appeared twice?